application/btsieve/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![warn(unused_extern_crates, rust_2018_idioms)]
```

Comment:
❓ why removing this warn?

Reply:
It is already included in `rust_2018_idioms` (check the commit message: https://github.com/comit-network/comit-rs/pull/874/commits/5d9bf694db170f84074667a80ce4d3f1a80f9fb7 ->
`#[warn(unused_extern_crates)] implied by #[warn(rust_2018_idioms)]`